src/ioctls/kvm_vec.rs

```diff
@@ -0,0 +1,426 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The ioctls module should keep the initial structure. Please checkout this comment: https://github.com/rust-vmm/kvm-ioctls/pull/12#discussion_r271179246 for details.

But since you are also adding some helpers that aren't exactly tiny, how about creating a sub-module in kvm-ioctls/ioctls called common or helpers or something to that extent. We can then add all other helpers there. What do you say?